libbeat/common/file/rotator.go

```diff
@@ -78,6 +96,10 @@ func NewFileRotator(filename string, options ...RotatorOption) (*Rotator, error)
 		return nil, errors.Errorf("file rotator permissions mask of %o is invalid", r.permissions)
 	}
 
+	if !isCompressionSupported(r.compression) {
+		return nil, errors.Errorf("Compression format '%v' is not supported", r.compression)
```

Comment:
Error strings should not start with a capital letter. https://github.com/golang/go/wiki/CodeReviewComments#error-strings